webapp/packages/plugin-data-spreadsheet-new/src/DataGrid/DataGridContextMenu/DataGridContextMenuFilter/DataGridContextMenuFilterService.ts

```diff
@@ -69,12 +68,6 @@ export class DataGridContextMenuFilterService {
     await model.request(() => {
       constraints.setFilter(resultColumn.position, operator, filterValue);
     });
-
-    const whereFilter = model.requestInfo.requestFilter || model.source.options?.whereFilter;
-
-    if (isNotNullDefined(whereFilter)) {
-      constraints.setWhereFilter(whereFilter);
```

Comment:
It will cause a regression, the constraints generated filter should be displayed in filter input